numpy/_core/meson.build

```diff
@@ -281,7 +281,7 @@ else
   # function is not available in CI. For the latter there is a fallback path,
   # but that is broken because we don't have the exact long double
   # representation checks.
-  if cc.get_argument_syntax() != 'msvc'
+  if not is_windows
```

Comment:
Hmm, this seems fine but really I should fix the feature check. Will try that.

Reply:
yeah, note that meson is using set10(), while the code just checks via"ifdef", which is doubly confusing :) I tried to not touch it.

Reply:
This check should be fixed by gh-24979. It would be useful if you could double check that that works for Mingw too.

Reply:
I've dropped this patch, now that it is no longer needed.